src/service/url-replacements-impl.js

```diff
@@ -195,7 +195,8 @@ export class GlobalVariableSource extends VariableSource {
 
     // Returns the title of this AMP document.
     this.set('TITLE', () => {
-      return this.ampdoc.win.document.title;
+      return this.ampdoc.win.document.originalTitle ||
+          this.ampdoc.win.document.title;
```

Comment:
Please add a comment like

The environment may override the title and set `originalTitle`. Prefer that if available.